### ISSUE TITLE
runInExternalNativeTerminal not working

### ISSUE THREAD

**Issue by dreambit** [firsttris/vscode-jest-runner#343]
runInExternalNativeTerminal not working, when i set it to true, test is still run in internal terminal

**Comment by firsttris** [comment]
can you provide which terminal and which os you are using.

**Comment by dreambit** [comment]
> can you provide which terminal and which os you are using.

Macos, Warp
<img width="515" alt="image" src="https://github.com/firsttris/vscode-jest-runner/assets/1479661/1ed9e98c-ec22-46c1-86f5-8563957a901e">

**Comment by andybonner** [comment]
This is true for me too, also on Mac OS; using native mac Terminal.app or my preferred iTerm.app. I also tried setting the "Terminal: Explorer kind" setting to "external" in case that made a difference; it didn't.

**Comment by dfnerio** [comment]
Hello! Any updates on this? I also use Wrap as my default/preferred terminal and this is pretty annoying...
